docs/data-sources/builtin-toolsets/servicenow.md

```
@@ -0,0 +1,156 @@
+        servicenow/tickets:
+          enabled: true
+          config:
+            verify_ssl: true
+            timeout: 30
+    ```
+
+## Advanced Configuration
+
+You can customize ServiceNow integration settings:
+
+```yaml
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)